code/src/App.js

```diff
@@ -1,9 +1,108 @@
-import React from 'react'
+import React, { useState } from "react";
+import { Summary } from "./components/Summary";
+import "./index.css";
+
+const selectHome = ["Downtown", "Suburb or small town", "Rural retreat"];
+const activityLevels = ["not very active", "moderately active", "very active"];
+const sizeGroups = ["tiny", "medium", "huge"];
 
 export const App = () => {
+  const [name, setName] = useState("");
+  const [reason, setReason] = useState("");
+  const [home, setHome] = useState("");
+  const [activity, setActivity] = useState("");
+  const [sizeGroup, setSize] = useState("");
+  const [showResult, setShowResult] = useState(false);
+
+  const handleSubmit = event => {
+    event.preventDefault();
+    setShowResult(true);
+  };
```

Comment:
Would be nice to catch an error here if the input values is empty so it doesn't always become `true`